caller-workflow-templates/cdeployment-ecs-backend-caller.yml

```diff
@@ -0,0 +1,81 @@
+name: CD pipeline for ECS backend
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push tag event
+  push:
+    tags:
+      - v*
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  VERSION: 'latest'
+
+jobs:
+  graasp-deploy-ecs-backend-workflow:
+    name: CD caller template
+    uses: graasp/graasp-deploy/.github/workflows/cdeployment-ecs.yml@main
+    with:
+      ecs-task-definition: '.aws/<name>-test.json'
+    secrets:
+      aws-access-key-id:
+      aws-secret-access-key:
+      aws-region:
+      ecs-cluster:
+      ecs-service:
+      container-name:
+      ecr-repository:
+      container-name-2:
+      container-name-3:
+      container-image-2:
+      container-image-3:
```

Comment:
Same as above. Not sure what these names refer to.

Reply:
Same as above,  ECS cluster for the graasp backend has 3 containers. This helps to keep container names and image names "secret".